### DISCUSSION TITLE
URL without Query Parameter

### DISCUSSION THREAD

**Discussion by ghost** [Infocaster/UrlTracker#178] [Q&A]
Hi, 

We send people to our site from different locations which we track using UTMs. If a page cannot be found a URL tracker will be created for each unique URL including the UTM in the query string.

I was wondering if there is functionality to record only the URL without the query parameters?

Any help would be much apreciated!

**Comment by D-Inventor** [comment]
Hi @Timmeeehh !! Thank you for reaching out. Though out-of-the-box the URL Tracker does not provide an option to discard query strings while registering these URLs, the URL Tracker should be sufficiently extendable to support your use case.

Try for example to implement a client error filter:
https://github.com/Infocaster/UrlTracker/wiki/Extending---Web#custom-client-error-filter

A client error filter allows you to decide which client errors you want to register and which ones you don't. I'm not sure, but I think you can also use this feature to modify the URL that is registered.

I hope this helps!!

Kind regards,
Dennis
